pom.xml

```diff
@@ -915,7 +916,7 @@
           <artifactId>maven-surefire-plugin</artifactId>
           <version>2.19.1</version>
           <configuration>
-            <argLine>-Djava.net.preferIPv4Stack=true</argLine>
+            <argLine>@{argLine} -Djava.net.preferIPv4Stack=true</argLine>
```

Comment:
what's the reason for needing this?

Reply:
This is needed because both surefire AND jacoco change the argLine so this is how you are supposed to configure it in order to allow both to update argLine